internal/runtime/runtime_test.go

```diff
@@ -39,24 +44,65 @@ import (
 var lc logger.LoggingClient
 
 const (
-	devID1     = "id1"
 	serviceKey = "AppService-UnitTest"
 )
 
+var testEvent = dtos.Event{
+	Versionable: commonDTO.NewVersionable(),
+	Id:          "82eb2e26-0f24-48aa-ae4c-de9dac3fb9bc",
+	DeviceName:  "FamilyRoomThermostat",
+	ProfileName: "Thermostat",
+	Created:     time.Now().Unix(),
+	Origin:      time.Now().Unix(),
+	Readings: []dtos.BaseReading{
+		{
+			Versionable:   commonDTO.NewVersionable(),
+			Id:            "82eb2e26-0f24-48aa-ae4c-de9dac3f1234",
+			Created:       time.Now().Unix(),
+			Origin:        time.Now().Unix(),
+			DeviceName:    "FamilyRoomThermostat",
+			ResourceName:  "Temperature",
+			ProfileName:   "Thermostat",
+			ValueType:     v2.ValueTypeInt64,
+			SimpleReading: dtos.SimpleReading{Value: "72"},
+		},
+	},
+	Tags: nil,
+}
+
+var testAddEventRequest = requests.AddEventRequest{
+	BaseRequest: commonDTO.BaseRequest{},
+	Event:       testEvent,
+}
+
+var testV1Event = models.Event{
+	ID:      "82eb2e26-0f24-48aa-ae4c-de9dac3fb9bc",
+	Device:  "FamilyRoomThermostat",
+	Created: time.Now().Unix(),
+	Origin:  time.Now().Unix(),
+	Readings: []models.Reading{
+		{
+			Id:        "82eb2e26-0f24-48aa-ae4c-de9dac3f1234",
+			Created:   time.Now().Unix(),
+			Origin:    time.Now().Unix(),
+			Device:    "FamilyRoomThermostat",
+			Name:      "Temperature",
+			ValueType: v2.ValueTypeInt64,
+			Value:     "72",
+		},
+	},
+	Tags: nil,
+}
+
 func init() {
 	lc = logger.NewMockClient()
 }
 
 func TestProcessMessageNoTransforms(t *testing.T) {
-	// Event from device 1
-
-	eventIn := models.Event{
-		Device: devID1,
-	}
-	eventInBytes, _ := json.Marshal(eventIn)
+	payload, _ := json.Marshal(testAddEventRequest)
```

Comment:
though the test variable is pre-seeded, it is a good practice to verify error in case some programming typo in terms of metadata.